motionplan/plannerOptions.go

```diff
@@ -39,14 +39,13 @@ const (
 	// default number of times to try to smooth the path.
 	defaultSmoothIter = 20
 
-	// names of constraints.
-	defaultLinearConstraintName         = "defaultLinearConstraint"
-	defaultPseudolinearConstraintName   = "defaultPseudolinearConstraint"
-	defaultOrientationConstraintName    = "defaultOrientationConstraint"
-	defaultObstacleConstraintName       = "defaultObstacleConstraint"
-	defaultSelfCollisionConstraintName  = "defaultSelfCollisionConstraint"
-	defaultRobotCollisionConstraintName = "defaultRobotCollisionConstraint"
-	defaultJointConstraint              = "defaultJointSwingConstraint"
+	// descriptions of constraints.
+	defaultLinearConstraintDesc         = "Constraint to follow linear path"
+	defaultPseudolinearConstraintDesc   = "Constraint to follow pseudolinear path, with tolerance scaled to path length"
+	defaultOrientationConstraintDesc    = "Constraint to maintain orientation within bounds"
+	defaultObstacleConstraintDesc       = "Collision between the robot and an obstacle"
+	defaultSelfCollisionConstraintDesc  = "Collision between components that are moving"
+	defaultRobotCollisionConstraintDesc = "Collision between different pieces of the robot"
```

Comment:
Want to further disambiguate these two:
"Collision between two robot components that are moving"
vs
"Collision between a robot component that is moving and one that is stationary"

Reply:
Yeah its tricky. I think components is probably the preferred word